src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/Worker.java

```diff
@@ -0,0 +1,16 @@
+package edu.kit.ipd.crowdcontrol.objectservice.crowdworking;
+
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Created by marcel on 19.01.16.
+ */
+public interface Worker {
+    /**
+     * Parse a worker id out of the params
+     * @param param The parameters which were sent by a platform
+     * @return The id of the worker if one can be found
+     */
+    Optional<String> identifyWorker(Map<String, String[]> param);
```

Comment:
i would throw an exception instead of using an optional. something went terribly wrong if the Platform is unable to parse it's own input.